test/NuGet.Core.FuncTests/Msbuild.Integration.Test/MsbuildRestoreTaskTests.cs

```diff
@@ -1547,102 +1510,6 @@ public async Task MsbuildRestore_PackageNamespaceLongerMatches_Succeed()
             }
         }
 
-        [PlatformFact(Platform.Windows)]
-        public async Task MsbuildRestore_PackageNamespaceLongerMatches_NoNamespaceMatchesLog()
```

Comment:
Same question as the one is NuGetRestoreCommandTest.cs  applies.

Reply:
@nkolev92 @kartheekp-ms 
Added back the tests with verification of a failure.